docs/FRSKY_R9.md

```diff
@@ -240,17 +242,19 @@ Wiring the receiver to the JR Bay on the back of your radio and flashing the ELR
 
 After flashing the ELRS bootloader, you can connect the serial port as described in the link above and use the ELRS UARTupload.py script to flash/update mLRS with latest mLRS firmware .elrs file. There are, of course, separate .elrs image files for each receiver:
 
+*Note: Python3 is a prerequisite to use the ELRS UARTUpload.py script.*
+
 1. Download the latest mLRS firmware or build with MLRS\_FEATURE\_ELRS\_BOOTLOADER defined.
 
 2. Download or git clone the [ExpressLRS](https://github.com/ExpressLRS/ExpressLRS) repository. Since you only need the src/python folder, if you have subversion installed, you could use the svn github interface to get just what you need. From the command line type something like this: "svn export https://github.com/ExpressLRS/ExpressLRS/trunk/src/python"
 
-3. Connect a TTL USB serial adapter to your computer and install a driver if it's not recognized. If you don't have a TTL USB serial adapter, you should be able to use your flight controller. See step 8.
+3. Connect a TTL USB serial adapter to your computer and install a driver if it's not recognized.  If you're having trouble flashing or don't have a TTL USB serial adapter, you should be able to use your flight controller - see step 8.
 
 4. Wire the serial adapter to your receiver as instructed [here](https://www.expresslrs.org/quick-start/receivers/r9/#wiring-up-your-receiver). But, don't connect the VCC wire yet.
 
 5. Use the command window to run the UARTupload.py script like: "python UARTupload.py image\_file\_path" where image\_file\_path is the path to the appropriate .elrs file you obtained in step 1.
 
-6. When the python script reports "attempting to reboot into bootloader", power up the receiver by connecting the VCC wire to the 5 volt output of your serial adapter or a 5 volt power supply.  You should see the script report sync and begin the firmware download. If it fails, try again, the timing can be a bit tight. The first time you flash the .elrs file, you may not need to delay connecting the VCC wire.  If you have trouble getting the timing correct, you can hold down the button when powering up the receiver and the ELRS bootloader will keep running and not start the application code.
+6. When the python script reports "attempting to reboot into bootloader", power up the receiver by connecting the VCC wire to the 5 volt output of your serial adapter or a 5 volt power supply.  You should see the script report sync and begin the firmware download. If it fails, try again, the timing can be a bit tight. The first time you flash the .elrs file, you may not need to delay connecting the VCC wire.  If you have trouble getting the timing correct, you can hold down the button when powering up the receiver and the ELRS bootloader will keep running and not start the application code. The red and green LEDs will blink alternately while the receiver is in the ELRS bootloader.
```

Comment:
With regards to the last sentence, I think I could be made stronger as regards the potential use of this information. Like e.g. "The red and green LEDs will blink alternately while the receiver is in the ELRS bootloader, which allows visual check if the step was succesful and the bootloader is running.". Or anything along this line.

Reply:
Reworked the last two sentences to make more sense.